### PR TITLE
[ModelingOutput] update modeling output and unittest

### PR DESCRIPTION
**Pull request by wj-Mcat**
### PR types
New features

### PR changes
Models

### Description

try to update ernie/roberta/roformer modeling output and related unittest

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,3 +10,4 @@ paddlefsl
 sentencepiece
 paddle2onnx
 protobuf>=3.1.0, <=3.20.0
+parameterized
```
Comment (guoshengCS) [inline requirements.txt]: 这个不作为安装依赖了吧，看看是否作为extras_require在测试时使用， @zjjlivein 也看下是否在CI单测时安装使用test的包

File: tests/transformers/bert/test_modeling.py [hunk 2]
```diff
@@ -25,35 +26,34 @@
...
+                 initializer_range=0.02,
+                 pad_token_id=0,
+                 pool_act="tanh",
+                 fuse=False,
+                 type_sequence_label_size=2,
+                 num_labels=3,
+                 num_choices=4,
+                 num_classes=3,
+                 scope=None,
+                 return_dict=False):
```
Comment (guoshengCS) [inline tests/transformers/bert/test_modeling.py:56]: 这里建议不增加return_dict参数，使用parent BertModelTest中的以减少后期更新调整工作

File: tests/transformers/bert/test_modeling.py [hunk 3]
```diff
@@ -197,15 +204,26 @@ def create_and_check_model_past_large_inputs(
...
-                                 return_dict=True)["hidden_states"][0]
+        outputs = model(next_input_ids,
+                        attention_mask=next_attention_mask,
+                        output_hidden_states=True,
+                        return_dict=self.return_dict)
+
+        if self.return_dict:
+            output_from_no_past = outputs['hidden_states'][0]
+        else:
+            output_from_no_past = outputs[2][0]
```
Comment (guoshengCS) [inline tests/transformers/bert/test_modeling.py]: 这个也跟其他case一样不加if self.return_dict判断吧

File: requirements.txt (hunk 1 above)
Comment (wj-Mcat) [reply requirements.txt]: 我这边之前的做法是：添加一个[`requirements-dev.txt`](https://github.com/wechaty/python-wechaty/blob/master/requirements-dev.txt)文件来提供在执行单测的一些特殊的依赖包。 仅供参考。

Comment (zjjlivein) [reply requirements.txt]: > 这个不作为安装依赖了吧，看看是否作为extras_require在测试时使用， @zjjlivein 也看下是否在CI单测时安装使用test的包

是的，这里不需要新增requirements.txt 文件，sentencepiece等在paddlenlp安装时已添加，parameterized等extras_require同步我在CI环境安装即可

Comment (wj-Mcat) [reply requirements.txt]: 嗯嗯，好的，那这部分我来调整一下

File: tests/transformers/bert/test_modeling.py (hunk 2 above)
Comment (wj-Mcat) [reply tests/transformers/bert/test_modeling.py:56]: 我在新的提交中做了如下调整：
1. 在Tester中删掉了`return_dict` 和 `use_labels` 参数的依赖。
2. Tester中的`return_dict`参数从self.parent.return_dict中取
3. `use_labels`根据`labels is None`来判断。

File: tests/transformers/bert/test_modeling.py (hunk 3 above)
Comment (wj-Mcat) [reply tests/transformers/bert/test_modeling.py]: 我在新的调整当中，都是通过索引来获取其指。

File: tests/transformers/bert/test_modeling.py [hunk 4]
```diff
@@ -164,9 +167,15 @@ def create_and_check_for_masked_lm(
         result = model(input_ids,
                        attention_mask=input_mask,
                        token_type_ids=token_type_ids,
-                       labels=token_labels)
+                       labels=token_labels,
+                       return_dict=self.return_dict)
+        if token_labels is not None:
+            result = result[1:]
+        elif paddle.is_tensor(result):
+            result = [result]
```
Comment (wj-Mcat) [inline tests/transformers/bert/test_modeling.py:175]: 1. use_labels 通过`token_labels`是否为空判断，这样就可以减少一个参数的依赖。
2. 有的模型是直接返回一个tensor，非tuple类型，故此时需要转化成数组来判断其shape大小。

File: tests/transformers/bert/test_modeling.py [hunk 5]
```diff
@@ -374,7 +404,7 @@ class BertModelTest(ModelTesterMixin, unittest.TestCase):
     )
 
     def setUp(self):
-        self.model_tester = BertModelTester(self)
+        self.model_tester = BertModelTester(self, return_dict=self.return_dict)
```
Comment (wj-Mcat) [inline tests/transformers/bert/test_modeling.py:407]: 在Tester中的`return_dict`直接从parent中获取了/

**Review by wj-Mcat — COMMENTED** [review]
Comment of Some Changes.

**Comment by wj-Mcat** [comment]
ping @guoshengCS

File: requirements.txt [hunk 6]
```diff
@@ -9,4 +9,4 @@ tqdm
 paddlefsl
 sentencepiece
 paddle2onnx
-protobuf>=3.1.0, <=3.20.0
+protobuf>=3.1.0, <=3.20.0
```
Comment (wj-Mcat) [inline requirements.txt:12]: 这里没有任何差别，可是还显示在diff files里面了。

**Review by wj-Mcat — COMMENTED** [review]
已经移除了parameterized的依赖。

**Review by guoshengCS — DISMISSED** [review]

**Comment by guoshengCS** [comment]
还要解决下冲突

**Review by guoshengCS — APPROVED** [review]
